src/qml/DigitizingToolbar.qml

```diff
@@ -219,4 +237,10 @@ VisibilityFadingRow {
     rubberbandModel.removeVertex()
     mapSettings.setCenter( rubberbandModel.currentCoordinate )
   }
+
+  function confirming()
```

Comment:
What's the diff between `confirming()` and `confirm()`?

Reply:
The function confirming() triggers code to finalize the digitizing, while signal confirm() is what's emitted when confirming is done. Not sure if that's helpful :) I hesitated between that and function confirm() signal confirmed().

Reply:
I think the second one would be clearer in terms of semantic api